docs/api/dandi-client.md

```diff
@@ -0,0 +1,66 @@
+# DANDI Client
+
+The DANDI Client is a Python library and command-line tool for interacting with the DANDI Archive. It provides functionality for downloading, validating, organizing, and uploading data to and from the DANDI Archive.
+
+## Installation
+
+You can install the DANDI Client using pip:
+
+```bash
+pip install dandi
+```
+
+## Documentation
+
+The full documentation for the DANDI Client is available at [https://dandi.readthedocs.io/](https://dandi.readthedocs.io/).
+
+## Key Features
+
+- Download data from the DANDI Archive
+- Validate NWB files
+- Organize data for upload to the DANDI Archive
+- Upload data to the DANDI Archive
+- Search for Dandisets and assets
```

Comment:
```suggestion
```

Reply:
The python library does provide search over dandisets:

https://dandi.readthedocs.io/en/latest/modref/dandiapi.html#dandi.dandiapi.DandiAPIClient.get_dandisets